### ISSUE TITLE
Feature Request: movepkg does not copy dependencies on AUR

### ISSUE THREAD

**Issue by cookiengineer** [trizen/trizen#233]
Hey there,

I am not sure whether this is expected behaviour, that's why I'm creating this as a Feature Request issue.

If you install a package on AUR via `trizen -S package-name` and it has a dependency that's also hosted on AUR, the dependency won't be moved to the `movepkg_dir`.

So, in the local package backup sense it is quite annoying if you don't have a copy of a package that's been installed as a dependency and want to reinstall it later.

I would love to help fixing this, but I'm unsure how the method call flow is, actually - because I've never created a Perl program. If I'm understanding it correctly, the `install_dependency` subroutine would need to have a `move_built_package` after the `install_package` call, right?

But how would you obtain the tarball name for that? Is the `$packages->{$name}` equivalent to the tarball path/filename?

**Comment by trizen** [comment]
Currently, if an AUR dependency is already installed, then it will not be rebuilt and moved. But if it's not installed, then it will be built and moved, along with the main package.

For example, this can be checked with:

```console
trizen -S sidef --movepkg-dir=/tmp/test
```

Then `ls /tmp/test` will show the AUR dependencies + the main package successfully moved in there.

So your request is to rebuild and move all the AUR dependencies? (even those that are already installed)

**Comment by cookiengineer** [comment]
Nope, what I'm saying that in my case when I do `trizen -S numix-circle-icon-theme-git`, then the upstream dependency of that package from AUR (`numix-icon-theme-git`) is not inside my `movepkg_dir` folder, only the directly installed package that was installed via `-S` CLI call.

**Comment by trizen** [comment]
Is the issue still reproducible with `trizen 1.64`? On my system, I cannot confirm this issue.

**Comment by cookiengineer** [comment]
I have tried it today on a plain new arch system. When both `movepkg` is set to `1` and `movepkg_dir` is set to `/var/cache/pacman/pkg`, it does not work with trizen `1.64-1`.

Am I missing an essential configuration flag here that I should check?

**Comment by trizen** [comment]
Strange. An alternative solution would be to set in `/etc/makepkg.conf`:

```ini
#-- Destination: specify a fixed directory where all packages will be placed
PKGDEST=/var/cache/pacman/pkg
```

(this probably requires the user to have read+write permissions in that directory)
